h/views/admin/groups.py

```diff
@@ -118,7 +116,7 @@ def on_success(appstruct):
             self.request.session.flash(
```

Comment:
There's like 15 lines of code here to create a link in the resulting flash message. I don't think the link provides a whole lot of value (and is available in the list of groups shown on the page you go to), and may pull this chunk in a subsequent PR.